coreneuron/io/phase2.cpp

```diff
@@ -482,12 +482,8 @@ void Phase2::set_net_send_buffer(Memb_list** ml_list, const std::vector<int>& pn
             ml->_net_receive_buffer = nrb;
             nrb->_pnt_offset = pnt_offset[type];
 
-            // begin with a size of 5% of the number of instances
+            // begin with a size equal to the number of instances
             nrb->_size = ml->nodecount;
-            // or at least 8
-            nrb->_size = std::max(8, nrb->_size);
-            // but not more than nodecount
-            nrb->_size = std::min(ml->nodecount, nrb->_size);
```

Comment:
Not sure if there was any strong reason to not have `non-zero` buffer size even if a mechanism is empty. (esp. keeping some min in order to avoid reallocation)

Reply:
You didn't get that anyway. If `ml->nodecount` was zero then `nrb->_size` was set to `0`, then `8`, then `0` again.

Reply:
yeah I saw that 🙈! (may be was gradually modified) But just saying that in case there was a reason, we could keep `std::max(8, ml->nodecount);`

Reply:
Done